source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -294,13 +325,34 @@ CGLTFMeshFileLoader::BufferOffset CGLTFMeshFileLoader::MeshExtractor::getBuffer(
 	return BufferOffset(buffer.data, view.byteOffset);
 }
 
+/**
+ * The index of the accessor that contains the vertex indices. 
+ * When this is undefined, the primitive defines non-indexed geometry. 
+ * When defined, the accessor MUST have SCALAR type and an unsigned integer component type.
+ * Documentation: https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#_mesh_primitive_indices
+ * Type: Integer
+ * Required: NO
+*/
 std::size_t CGLTFMeshFileLoader::MeshExtractor::getIndicesAccessorIdx(
 		const std::size_t meshIdx,
 		const std::size_t primitiveIdx) const
 {
 	return m_model.meshes[meshIdx].primitives[primitiveIdx].indices;
 }
 
+/**
+ * The datatype of the accessor’s components.
+ * UNSIGNED_INT type MUST NOT be used for any accessor that is not referenced by mesh.primitive.indices.
+ * Documentation: https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#_accessor_componenttype
+ * Type: Integer
+ * Required: YES
+*/
+int CGLTFMeshFileLoader::MeshExtractor::getComponentType(const std::size_t accessorIdx) const {

```

Comment:
Where is this method used? I think there may be a method that already does this in tinygltf.

Reply:
Let me gut it out then